openedx/core/djangoapps/coursegraph/management/commands/tests/test_dump_to_neo4j.py

```diff
@@ -154,11 +138,13 @@ def test_coerce_types_iterable(self, iterable_type):
         (b"plain string", b"plain string"),
         (True, True),
         (None, "None"),
+        ((1,), "(1,)"),
+        ([object, object], ["<type 'object'>", "<type 'object'>"])
```

Comment:
@adampalay This looks good, but I would suggest, to add a small comment here, which kind of data we are passing.

Just a suggestion Looks good to me 👍